### PR TITLE
feat(behavior_velocity,behavior_path,route_handler): use route handler in behavior velocity

### PR DESCRIPTION
**Pull request by taikitanaka3**
## Related Issue(required)

#361

## Description(required)

- use setMap function in behavior velocity
- some functions in route handler will be used in occlusion spot at another PR(add detail later).

## Review Procedure(required)

launch psim and see each modules works the same as before

Screenshot from 2022-03-03 12-57-29

@yukkysaito please :+1: if this change looks good

## Related PR(optional)

- [x] release note

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by takayuki5168** [comment]
LGTM

File: planning/route_handler/src/route_handler.cpp [hunk 1]
```diff
@@ -1271,11 +1271,26 @@ lanelet::ConstLanelets RouteHandler::getCheckTargetLanesFromPath(
...
+bool RouteHandler::isMapMsgReady() const { return is_map_msg_ready_; }
+
+lanelet::routing::RoutingGraphPtr RouteHandler::getRoutingGraphPtr() const
+{
+  return routing_graph_ptr_;
+}
+
+lanelet::traffic_rules::TrafficRulesPtr RouteHandler::getTrafficRulesPtr() const
 {
-  return *overall_graphs_ptr_;
```
Comment (taikitanaka3) [inline planning/route_handler/src/route_handler.cpp:1276]: replaced unsafe function to return pointer to function to return shared_ptr  itself

File: planning/behavior_velocity_planner/src/node.cpp [hunk 2]
```diff
@@ -252,32 +255,7 @@ void BehaviorVelocityPlannerNode::onLaneletMap(
...
-      TrafficRulesFactory::create(Locations::Germany, Participants::Pedestrian);
-
-    RoutingGraphConstPtr vehicle_graph =
-      RoutingGraph::build(*planner_data_.lanelet_map, *traffic_rules);
-    RoutingGraphConstPtr pedestrian_graph =
-      RoutingGraph::build(*planner_data_.lanelet_map, *pedestrian_rules);
-    RoutingGraphContainer overall_graphs({vehicle_graph, pedestrian_graph});
-
-    planner_data_.overall_graphs = std::make_shared<const RoutingGraphContainer>(overall_graphs);
-  }
```
Comment (taikitanaka3) [inline planning/behavior_velocity_planner/src/node.cpp:280]: replaced same function in route handler

**Review by rej55 — APPROVED** [review]
I confirm the simulation runs.
LGTM.

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**